client/patch-react.js

```diff
@@ -19,6 +19,18 @@ export default (handleError = () => {}) => {
       const { prototype } = Component
       if (prototype && prototype.render) {
         prototype.render = wrapRender(prototype.render)
+      } else if (prototype && prototype.constructor) {
+        // Still a React component instance, but there's no render method in
```

Comment:
For now, we don't monkeypatch `componentWillMount` if `prototype.render` exists.